tla/tlc.sh

```diff
@@ -3,7 +3,17 @@
 # Licensed under the Apache 2.0 License.
 # Original License below
 # Adapted from: https://github.com/pmer/tla-bin
-exec java -cp tla2tools.jar:CommunityModules-deps.jar tlc2.TLC "$@" -workers auto
+
+set -x
+
+if [ "${CI}" ]; then
+    JVM_OPTIONS=("-Dtlc2.TLC.ide=Github" "-Dutil.ExecutionStatisticsCollector.id=be29f6283abeed2fb1fd0be898bc6601")
+    TLC_OPTIONS=(-checkpoint 60)
+else
+    TLC_OPTIONS=(-coverage 60)
```

Comment:
If you'd like to simplify things, I think it's fine to remove this. I think most people using local dev will be using the vscode plug-in

Reply:
On second thoughts, I think its best to remove this. The coverage info when printed to the stdout is kinda hard to read and might confuse beginners

Reply:
Agreed, taking it off.